RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronID_cff.py

```diff
@@ -11,10 +11,20 @@
     )
 
 from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
-run2_miniAOD_UL.toModify(
+from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel
+from Configuration.Eras.Modifier_bParking_cff import bParking
+( run2_miniAOD_UL | bParking ).toModify(
```

Comment:
we actually need `UL | (b & UL)` here, which is the same as UL
```suggestion
run2_miniAOD_UL.toModify(
```

without this there is still a difference in reco, as can be seen in 136.898
![all_jenkins33589-5690ec-15677VS03-1100-jenkins33962-cbafaf-15636_RunParkingBPH2018Bwf136p898c_floatedmValueMap_lowPtGsfElectronID__reRECO_obj_values_](https://user-images.githubusercontent.com/4676718/120901990-b0914500-c5f2-11eb-9356-e527a2bef3e0.png)